src/App.js

```diff
@@ -10,49 +10,50 @@ class App extends Component {
           list: [],
           inputValue: ''
       }
-
-      this.addHandler = this.addHandler.bind(this);
-      this.changeInputHandler = this.changeInputHandler.bind(this);
     }
 
-    addHandler() {
-        let newObj = {
-            text: this.state.inputValue
-        };
-
-        console.log(newObj);
-
-        this.setState({
-            list: [...this.state.list, newObj],
-            inputValue: ''
-        });
-
-        console.log(this.state.list);
+    onClickAddItem = () => {
+        if(this.state.inputValue.length > 0) {
```

Comment:
you can 
```suggestion
        if(this.state.inputValue.length) {
```
since if statement will try to convert any value inside it and any value that greater than 0 is true but 0 is false